compiler/rustc_ast/src/node_id.rs

```diff
@@ -2,6 +2,12 @@ use rustc_span::ExpnId;
 use std::fmt;
 
 rustc_index::newtype_index! {
+    /// Identifies an AST node.
+    ///
+    /// This identifies top-level definitions, expressions, and everything in between.
+    /// This is later split into [`DefId`] and `HirId` for the HIR.
```

Comment:
Could you say "converted" instead of "split"?

Reply:
"converted" didn't sound quite right to me, so I did "turned into" instead.